content/en/user-guide/tools/cloud-pods/launchpad/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Cloud Pods Launchpad"
+weight: 10
+categories: ["LocalStack Pro", "Tools"]
+description: >
+  The LocalStack Cloud Pods Launchpad offers the easiest way of sharing and injecting cloud pods directly from your repositories or via a simple URL.
+---
+
+## Overview
+For more general information about cloud pods, please visit the [Cloud Pods Overview]({{< ref "cloud-pods" >}}). <br>
+The LocalStack Cloud Pods Launchpad allows you to quickly share and inject cloud pods into a LocalStack instance.
+
+## Creating your sharable link
+Navigate to [app.localstack.cloud/launchpad](https://app.localstack.cloud/launchpad) where you can create a sharable link for your pods.
```

Comment:
Clarification question: is the badge hard-coded, right? What if we want to have multiple badged in a repo? How can we distinguish them?

Reply:
As of right now the badges would be identical - there's no good way of customizing these without relying on shield.io's creation.

Reply:
How would you like to customize/distinguish between the badges? @giograno

Reply:
In case we want to add dynamically labeled badges, I think we should try to get our logo into [simple icons](https://github.com/simple-icons/simple-icons), to be used as outlined in shields.io docs here: https://github.com/badges/shields/blob/master/doc/logos.md#simpleicons

also @thrau